dask/array/core.py

```diff
@@ -5190,4 +5182,5 @@ def new_da_object(dsk, name, chunks, meta=None, dtype=None):
         return Array(dsk, name=name, chunks=chunks, meta=meta, dtype=dtype)
 
 
-from .utils import meta_from_array, compute_meta
+from .blockwise import blockwise
```

Comment:
I moved this to the bottom of the file to avoid circular dependencies. This is the only change that I made to what `isort` did by default. @jrbourbeau